examples/charts/toolpad/pages/page/page.yml

```
@@ -0,0 +1,96 @@
+apiVersion: v1
+kind: page
+spec:
+  id: tffH0lV
+  title: Default page
+  content:
+    - component: PageRow
+      name: pageRow
+      props:
+        justifyContent: start
+      children:
+        - component: Chart
+          name: chart
+          props:
+            data:
+              - label: No. of movies released per year
+                kind: line
+                data:
+                  $$jsExpression: |
+                    query1.data.map((movie) => ({
+                      year: movie.year,
+                      count_id: query1.data.reduce((acc, currentMovie) => {
+                        return currentMovie.year === movie.year ? acc + 1 : acc
+                      }, 0),
+                    }))
+                color: '#f44336'
+                xKey: year
+                yKey: count_id
+        - component: Chart
+          name: chart1
+          props:
+            data:
+              - label: No. of movies released per year
+                kind: bar
+                data:
+                  $$jsExpression: |
+                    query1.data.map((movie) => ({
+                      year: movie.year,
+                      count_id: query1.data.reduce((acc, currentMovie) => {
+                        return currentMovie.year === movie.year ? acc + 1 : acc
+                      }, 0),
+                    }))
+                color: '#673ab7'
+                xKey: year
+                yKey: count_id
+    - component: Chart
+      name: chart2
+      props:
+        data:
+          - label: No. of movies released per year
+            kind: area
+            data:
+              $$jsExpression: |
+                query1.data.map((movie) => ({
+                  year: movie.year,
+                  count_id: query1.data.reduce((acc, currentMovie) => {
+                    return currentMovie.year === movie.year ? acc + 1 : acc
+                  }, 0),
+                }))
+            color: '#00bcd4'
+            xKey: year
+            yKey: count_id
+      layout:
+        columnSize: 1
+    - component: Chart
+      name: chart3
+      props:
+        data:
+          - label: No. of movies released per year
+            kind: scatter
+            data:
+              $$jsExpression: |
+                query1.data.map((movie) => ({
+                  year: movie.year,
```

For the scatter chart to show in this case:
```suggestion
                  year: Number(movie.year),
```